crates/holochain_sqlite/src/conn.rs

```diff
@@ -0,0 +1,140 @@
+use crate::prelude::*;
+use chashmap::CHashMap;
+use lazy_static::lazy_static;
+use rusqlite::*;
+use std::{
+    path::{Path, PathBuf},
+    time::Duration,
+};
+
+mod singleton_conn;
+
+const SQLITE_BUSY_TIMEOUT: Duration = Duration::from_secs(30);
+
+lazy_static! {
```

Comment:
Prefer `once_cell::sync::Lazy` because it's more efficient and doesn't require a macro.